src/app/cube/home/components/collections/collections.component.html

```diff
@@ -13,7 +13,10 @@
   </div>
   <div class="collections__grid">
     <div *ngFor="let c of collections">
-      <clark-collection-card [collection]="c"></clark-collection-card>
+      <div class="card-wrapper" [ngClass]="{'center': c.abvName === 'plan c'}">
```

Comment:
I think we need to add some kind of marker to tie it in to the notice at the bottom. Like and asterisk or a badge. The two seem a little disconnected.

Reply:
With asterisk:
<img width="892" alt="Screen Shot 2020-03-16 at 9 55 22 AM" src="https://user-images.githubusercontent.com/32078831/76765125-49c50900-676c-11ea-8211-c5211be011dc.png">

Reply:
How would you think the badge would look?